docs/topics/installing-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-extension_{context}"]
+= Installing the {ProductShortName} extension for IntelliJ IDEA
+
+You can install the {ProductShortName} extension in the Ultimate and the Community Edition releases of IntelliJ IDEA.
+
+.Prerequisites
+include::snippet_jdk-hardware-mac-prerequisites.adoc[]
+
+* The latest version of `mta-cli` from the link:{MTADownloadPageURL}[{ProductShortName} download page]
+
+.Procedure
+
+. In IntelliJ IDEA, click *Plugins* on the Activity bar.
+. Enter `{ProductName}` in the Search field of the *Marketplace^ tab.
```

Comment:
```suggestion
. Enter *`{ProductName}`* in the Search field of the *Marketplace* tab.
```
For MMS, do you use bold to indicate text that the user enters? If so, then this should be bold.

Reply:
User input is always entered `input` -- good catch on Marketplace.

https://redhat-documentation.github.io/asciidoc-markup-conventions/  (See "Inline operations and user input")

Reply:
Excellent. I love that you're schooling me.

Reply:
It's about time I returned the favor :)